cfme/tests/configure/test_access_control.py

```diff
@@ -2835,3 +2837,52 @@ def test_tenant_template_visibility():
         initialEstimate: 1/10h
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@test_requirements.multi_tenancy
+@pytest.mark.meta(coverage=[1650484])
+def test_check_provider_count():
+    """
+        after adding one cloud provider it should show only one provider name in list
+    Bugzilla:
+        1650484
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/4h
+        testSteps:
+            1.Navigate to compute -> Cloud -> Tenants
+            2.Go to Configuration -> Create cloud tenant
+        expectedResults:
+            1.
+            2.It should show only one provider name.
+
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@test_requirements.multi_tenancy
+@pytest.mark.meta(coverage=[1654718])
+def test_default_tenat_name_rail_console():
+    """
+        Default tenent nme should be updated after setting up it
+    Bugzilla:
+        1654718
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/4h
+        testSteps:
+            1.Set a new default tenant name, e.g. 'Bit63'
+            2.From Rails console, lookup the corresponding tenant object, i.e.
+        expectedResults:
+            1.
+            2.tenant name should be updated on rail console
```

Comment:
```suggestion
            1. Set a new default tenant name, e.g. 'Bit63'
            2. From Rails console, lookup the corresponding tenant object, i.e.
        expectedResults:
            1.
            2. tenant name should be updated on rail console
```
Some additional about the commands used in the rails console would be helpful here.